refarch-eai/src/main/resources/application-local.yml

```
@@ -2,4 +2,12 @@ server:
   port: 8085
 
 logging:
-  include-application-group: false
+  include-application-group: false # disabled because application group is only relevant when deployed
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)